README.md

```diff
@@ -1,35 +1,35 @@
-Ghub.el — Minuscule client library for the Github API
+Ghub.el — Miniscule client library for the GitHub API
 =====================================================
 
-Ghub is a library that provides basic support for using the Github
+Ghub is a library that provides basic support for using the GitHub
 REST (v3) and GraphQL (v4) APIs from Emacs packages.  It abstracts
 access to API resources using only a handful of functions that are
 not resource-specific.
 
-It also handles the creation, storage and use of access tokens using a
-setup wizard, to make it easier for users to get started and to reduce
-the support burden imposed on package maintainers.  It also comes with
+It also handles the creation, storage, and use of access tokens by using a
```

Comment:
I did some research about this some time ago and decided to go with the Oxford comma even though it feels very strange (we don't use that in German). Two or three years later it still feels very strange and I have decided to go back to the way that feels natural to me (and also is used by many native English speakers).

I think `by` is optional here.

Reply:
> many native English speakers

Never assume the majority pays attention to this stuff, though :-) I think everyone knows that oxford-comma is *correct*; it's just that not everyone *cares*. Such is language.